File: PS1/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void) {
    int height = get_int("Height : ");
    while(height<=0){
        height = get_int("Height: ");
    }
    for (int row = 1; row <= height; row++) {
        for (int space = 0; space < height - row; space++) {
            printf(" ");
        }
        for (int hash = 0; hash < row; hash++) {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: PS6/sentimental-mario-less/mario.py
from cs50 import get_int
Height = get_int("Height: ")
while Height not in range(1, 9):
    Height = get_int("Height: ")

for row in range(1, Height + 1):
    for space in range(Height - row):
        print(" ", end="")
    for hash_symbol in range(row):
        print("#", end="")
    print()
